proptest/src/test_runner/failure_persistence/file.rs

```diff
@@ -253,36 +267,60 @@ fn parse_seed_line(
     mut line: String,
     path: &Path,
     lineno: usize,
-) -> Option<PersistedSeed> {
+) -> Option<(PersistedSeed, PersistedEdgeBias)> {
     // Remove anything after and including '#':
     if let Some(comment_start) = line.find('#') {
         line.truncate(comment_start);
     }
 
     if line.len() > 0 {
-        let ret = line.parse::<PersistedSeed>().ok();
-        if !ret.is_some() {
-            eprintln!(
-                "proptest: {}:{}: unparsable line, ignoring",
-                path.display(),
-                lineno + 1
-            );
+        let seed;
+        let mut edge_bias: PersistedEdgeBias = 0f32.to_le_bytes();
+        if let Some(comma_position) = line.find(',') {
+            seed = line[0..comma_position].parse::<PersistedSeed>().ok();
+            // The result is safe to ignore to not spam the log in case old
+            // cases exist, in which case edge_bias will be 0 to match old
+            // behaviour.
```

Comment:
a little confused here, wouldn't the `,` already be the token dictating whether or not there's an edge bias?

also `,` is a fairly arbitrary delimeter. if we're extending the seed parsing maybe we can choose something more explicit so it doesn't conflict with future metadata someone may wish to add? how about `@e=` as the token, `@` is not base-16 so a clear delimeter that this is new metadata, and `e=` meaning "edge equals". thoughts?

Reply:
It might take me a while to remember what I wrote over a year ago, so I'm sorry if I make any mistakes here.

I think the idea wasn't to indicate that edge bias was used, but to handle and store the seed for the edge bias separately, so that the old persistence files would be compatible. Right now I'm not entirely sure if that's necessary, might need some more investigation/thinking.